Holovibes/sources/core/holovibes.cc

```diff
@@ -91,7 +91,8 @@ void Holovibes::init_record_queue()
         }
 
         auto record_fd = gpu_output_queue_.load()->get_fd();
-        record_fd.depth = record_fd.depth == 1 ? 2 : record_fd.depth;
+        if (static_cast<int>(record_fd.depth) == 1)
```

Comment:
Skill issue. Replace by bits8 enum instead of casting to int

Reply:
Fixed.